data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+				to display
+					has "Passive-Aggressive Observations: done"
+
+
+			`	To the side of the table, you notice a model of the Mercury-like planet you landed on here in their home system, though strangely it doesn't come up on the screen at any point.`
+				to display
+					has "Brief Bystander: done"
+
+
+			`	An image of two distinct ships comes up, matching a pair of models on the table. One of them is the same type of Vi ship you first came across in the sand on a desert world, and the other is a Scin ship that looks like a derivative of other Gegno ships. You hear someone in the crowd yell loudly in what might be disappointment, but they are silenced by the Gegno trio.`
+				to display
+					has "Foreign Spectator I: failed"
+
+
+			`	One of the more heavily represented situations is the one that involved a large battle between the Vi and Scin over a moon orbiting a forest world. A long list of phrases scrolls by with graphics of different Gegno ships next to them, probably those who took part in said battle. The Gegno around you murmur to each other, occasionally grunting.`
+				to display
+					has "Battle over Zydee: offered"
+
+
+			`	On the far end of the table, you see a small model of an ice moon, but besides that, nothing else comes up regarding it.`
+				to display
+					has "Scin on Cyife: offered"
+
+			`	You look up to see the trio of Gegno still intently staring at you, meanwhile all of others surrounding you are glaring at you intensely. On the display, two icons appear side by side, and under each of them is a long list of phrases. As they scroll, the Gegno break out in discussions with one another, some loud and some not. You almost jump when the trio address the crowd and begin a speech of sorts, sometimes gesturing to the crowd, and sometimes gesturing to you. Although most of the Gegno seem frustrated, the trio is able to calm the crowd.`
+				to display
+					or
+						"reputation: Gegno Vi" < 0
+						"reputation: Gegno Scin" < 0
+
+			`	The bulkiest Gegno of the trio then places a model of a massive Vi warship in the middle of the table, one that you definitely recognize. More angry outbursts come from the surrounding Gegno, and while the one who placed the model glares at you, the other two eventually deescalate the uproar.`
+				to display
+					has "dead augen"
+
+
+			`	After having gone through all of the cases which involved you, the trio addresses the gathering once more with a big speech. Their voices echo through the outside air, and all of the Gegno respond with concurrent bellows or chants, followed by more silence. Oddly enough, you've been treated as an equal with all of the Gegno here. You suddenly feel as if you are standing amongst them as part of the gathering, rather than as some foreign alien. You aren't restricted at all, even with a weapon on you.`
```

Comment:
Is the "weapon on me" my sidearm, or a guard aiming a gun at me?

Reply:
(It's the sidearm you brought with you)